src/speed.c

```diff
@@ -0,0 +1,134 @@
+#include "speed.h"
+
+/**
+ * Maximum acceleration and deceleration.
+ *
+ * - Linear acceleration is defined in meters per second squared.
+ * - Linear deceleration is defined in meters per second squared.
+ * - Angular acceleration is defined in radians per second squared.
+ */
+static volatile float linear_acceleration;
+static volatile float linear_deceleration;
+static volatile float angular_acceleration;
+static volatile float max_linear_speed;
+static volatile float max_angular_speed;
+static volatile float turn_linear_speed;
+static volatile float turn_radius;
+static volatile float t0;
+static volatile float t1;
+
+const float linear_acceleration_defaults[NUM_MODES] = {5., 0., 0.};
+const float linear_deceleration_defaults[NUM_MODES] = {5., 0., 0.};
+const float angular_acceleration_defaults[NUM_MODES] = {32. * PI, 0., 0.};
+const float max_linear_speed_defaults[NUM_MODES] = {.5, 0., 0.};
+const float max_angular_speed_defaults[NUM_MODES] = {3 * PI, 0., 0.};
+const float turn_linear_speed_defaults[NUM_MODES] = {0.448, 0., 0.};
+const float turn_radius_defaults[NUM_MODES] = {0.11, 0., 0.};
```

Comment:
Default should be `0.07.` I think, not `0.11`.

Reply:
:+1: